### ISSUE TITLE
Some early feedback

### ISSUE THREAD

**Issue by snan** [mudita/mudita-center#577]
I'm gonna leave some feedback here, instead of writing a review on the forum or my blog, as if this was a bug report on a work-in-progress app rather than a consumer review for a shipped app.

I'm glad backups and OS updates work already. That's awesome.

Most frustrating part is that I can't **add custom text message templates**, neither in the Mudita Center nor on the device. That's a feature I'm really looking forward to. Both to deal with texting in general while on the go "I'll email you when I get home" or "Can you call me?" instead of letter-by-letter-ing it out on the number pad, and also as a workaround to be able to send my fave emojis. (I was happy to see that templates already append to the message, rather than replace it, so you can have a template for a hard-to-write name using foreign diacritics, or for a favourite emoji. That's gonna work great.)

I was testing out the messaging app.

First, theres' a bug: new messages show up from my friend in the list of senders on the left but not in the actual open conversation thread view itself, not until after a while, after there has been more messages from me.

Second, there's no way to copy out messages from him (so that I can paste them into an app with better emoji support, to view his emojis). Yeah, yeah, I could make a backup and then open the back up files in Emacs or something, but there's no easy way to copy messages out.

Third, the message length is a bit frustrating, both intrinsically (many other featuree phones I've had support way longer messages, by concatenating messages under the hood and, worst case, converting them to MMS messages) and in the UX, since Ctrl-V fails when trying to paste text that would make the message overlong.

Fourth, the latency and slowness of Electron apps even on this new super machine is hard to get used to. But that's on Electron, not you.

**Comment by snan** [comment]
Additionally, I was having trouble connecting initially, and by digging through the Mudita Center's logs I found the issue was [groups and permissions related, and easily resolvable](https://forum.mudita.com/t/mudita-center-on-debian/3382) (once I knew what was going on, which was the difficult part). One of the suggestions there says "use the provided USB-C cable" but that's something you might wanna replace, since there is no cable provided in the box. Which is a good thing (stop e-waste), but makes the message wrong.

Also, whenever I start typing my email address, before I even get get to the @ sign, it glares red "invalid email address". That's a bit counter to the chill yoga vibes you're otherwise going for. The warning, of course, disappears once I've typed the whole address.

**Comment by snan** [comment]
Thanks for having patience with me typing a bunch of issues into one issue on here. Please break it down into separate Jira tickets as you see fit. ♥

**Comment by snan** [comment]
Also I can't upload music and such.

**Comment by dkarski** [comment]
@snan I'd like to convey that many of the improvements you've mentioned have already been addressed and are visible in the latest release of the Mudita Center. Apologies for the delay in response; we've neglected the handling of issues on GitHub. If you have any further questions or concerns that haven't been resolved yet, please create a new issue based on the latest version of the Center, and I'll strive to respond promptly.

**Comment by dkarski** [comment]
Closing this issue is prompted by the resolution of the problems outlined in the message. Kindly open a new issue for any further questions or issues. Thank you.

**Comment by snan** [comment]
> One of the suggestions there says "use the provided USB-C cable" but that's something you might wanna replace, since there is no cable provided in the box. 

I actually just the other day found this cable. It was inside of the cardboard lining.
